TombEngine/Renderer/RendererDraw.cpp

```diff
@@ -1623,6 +1623,29 @@ namespace TEN::Renderer
 		PrepareDynamicLight(dynamicLight);
 	}
 
+	void Renderer::AddDynamicFogBulb(const Vector3& pos, float radius, float density, const Color& color, int hash)
```

Comment:
Probably it can be good idea to also expose `EmitFogBulb` method in lua API.

Reply:
fixed